<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCLogDlg</class>
 <widget class="QWidget" name="RCLogDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查看日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(69, 76, 120);
color: rgb(255, 255, 255);
border: none;
border-radius: 2px;
width:60px;

}
 
QPushButton:hover{
background-color: rgb(0, 0, 152);
color: rgb(255, 255, 255);
border: none;
border-radius: 3px;

}
 
QPushButton:pressed{
background-color: rgb(0, 0, 152);
color: rgb(255, 255, 255);

}
</string>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(69, 76, 120);
color: rgb(255, 255, 255);
border: none;
border-radius: 2px;
width:60px;

}
 
QPushButton:hover{
background-color: rgb(0, 0, 152);
color: rgb(255, 255, 255);
border: none;
border-radius: 3px;

}
 
QPushButton:pressed{
background-color: rgb(0, 0, 152);
color: rgb(255, 255, 255);

}
</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="logEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
